<launch>
    <arg name="gpu_device_id" default="0"/>
    <arg name="score_threshold" default="0.30"/>
    <arg name="nms_threshold" default="0.30"/>

    <arg name="network_definition_file" default="$(find vision_darknet)/darknet/cfg/yolov3.cfg"/>
    <arg name="pretrained_model_file" default="$(find vision_darknet)/darknet/data/yolov3.weights"/>
<!--
    <arg name="camera_id" default="/"/>
    <arg name="image_src" default="/image_raw"/>

    <include file="$(find zed_wrapper)/launch/zed.launch" />
-->
    <node pkg="vision_darknet" name="vision_darknet_detect" type="vision_darknet_detect" output="screen">
    <!--
        <remap from="/zed/left/image" to="/zed/left/image_rect_color" />
        <remap from="/zed/right/image" to="/zed/right/image_rect_color" />
    -->
        <param name="viz" type="bool" value="true" />
        <param name="network_definition_file" type="str" value="$(arg network_definition_file)"/>
        <param name="pretrained_model_file" type="str" value="$(arg pretrained_model_file)"/>
        <param name="score_threshold" type="double" value="$(arg score_threshold)"/>
        <param name="nms_threshold" type="double" value="$(arg nms_threshold)"/>
        <param name="gpu_device_id" type="int" value="$(arg gpu_device_id)"/>
       <!-- 
        <param name="image_raw_node" type="str" value="$(arg camera_id)$(arg image_src)"/>
       -->
    </node>

</launch>


